src/main/resources/org/spongepowered/launch/ivysettings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<ivysettings>
+    <settings defaultResolver="sponge" />
+    <resolvers>
+        <chain name="sponge">
+			<ibiblio name="m2" m2compatible="true" />
```

Comment:
the forge repo keeps all the libraries compressed we have the raw uncompressed versions in our repo